OneCommunityGlobal/HighestGoodNetworkApp
# Lintfixes for Badge Component

# Description
Lint fixes for Badge Component

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin user
4. go to Other Links→ Badge Management
5. set up ESLint and Prettier in VSCode
6. Check if the files in src/component/Badge/** doesn't have any errors

## Note:
Please let me know, if something needs to be modified, or can be optimized
